src/ConfigProvider.php

```diff
@@ -39,21 +27,18 @@ public function getDependenciesConfig(): array
             'abstract_factories' => [
                 FormAbstractServiceFactory::class,
             ],
-            'aliases' => [
-                'Laminas\Form\Annotation\FormAnnotationBuilder' => 'FormAnnotationBuilder',
-                AnnotationBuilder::class => 'FormAnnotationBuilder',
+            'aliases'            => [
                 FormElementManager::class => 'FormElementManager',
             ],
-            'factories' => [
-                'FormElementManager' => FormElementManagerFactory::class,
-                'FormAnnotationBuilder' => AnnotationBuilderFactory::class,
-            ]
+            'factories'          => [
+                FormElementManager::class => FormElementManagerFactory::class,
```

Comment:
```suggestion
                'FormElementManager'      => FormElementManagerFactory::class,
                FormElementManager::class => FormElementManagerFactory::class,
```
Without this line, it throws the following exception:

> Unable to resolve service "FormElementManager" to a factory; are you certain you provided it during configuration?